src/main/cpp/t06_max_count.cpp

```diff
@@ -30,5 +30,18 @@
 using namespace std;
 
 int t06_max_count() {
-
+int N=-1, M=0, x;
+cin >> x;
+while (x !=0)
+{
+    if(x > N)
+    {
+        N = x;
+        N = 1;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'N' is reassigned a value before the old one has been used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4592457)